src/tests/Common/CoreCLRTestLibrary/CoreClrConfigurationDetection.cs

```diff
@@ -30,13 +36,11 @@ public static class CoreClrConfigurationDetection
     public static bool IsStressTest =>
         IsGCStress ||
         IsDisableR2R ||
-        IsTailCallStress ||
-        IsJitStressRegs ||
-        IsJitStress ||
-        IsJitMinOpts;
+        IsAnyJitStress ||
+        IsHeapVerify;
 
-    private static string GetEnvironmentVariableValue(string name) =>
-        Environment.GetEnvironmentVariable("DOTNET_" + name) ?? Environment.GetEnvironmentVariable("COMPlus_" + name) ?? "0";
+    private static string GetEnvironmentVariableValue(string name, string defaultValue = "0") =>
```

Comment:
hm, this might warrant a comment in clrconfigvalues to make sure defaults get reflected here.

Reply:
I don't expect us to add many more cases here, if any (I'm only adding cases that we currently disable a ton of tests across).